test/operator/operator_test.py

```diff
@@ -213,7 +191,9 @@ def test_linear_Op():
     x = np.random.rand(3)
     out = np.random.rand(3)
 
-    Aop = MultiplyOp(A)
+    r3 = odl.Rn(3)
+
+    Aop = MatVecOperator(r3, r3, A)
```

Comment:
A note for later is that this syntax for `MatVecOperator` isnt optimal. I think automatic deduction of domain and range is very nice, especially when writing tests.